client-src/elements/chromedash-all-features-page.js

```diff
@@ -48,12 +48,12 @@ export class ChromedashAllFeaturesPage extends LitElement {
       this.query = this.rawQuery['q'];
     }
 
-    if (this.rawQuery.hasOwnProperty('start')) {
-      this.start = this.rawQuery['start'];
+    if (this.rawQuery.hasOwnProperty('start') && !Number.isNaN(+this.rawQuery['start'])) {
+      this.start = +this.rawQuery['start'];
```

Comment:
Can we use `parseInt()` here rather than `+`?
Some future maintainer might not understand why there is a `+` there.